src/github/entities/allowed-organization.entity.ts

```diff
@@ -0,0 +1,34 @@
+import { OrganizationStatus } from 'src/common/constants';
+import { Base } from 'src/universal/base.entity';
+import { Column, Entity } from 'typeorm';
+
+@Entity()
+export class AllowedOrganization extends Base {
+  @Column({ type: 'varchar', length: 255, default: 'github' })
+  host: string;
+
+  @Column({ type: 'varchar', length: 255, unique: true })
+  organizationName: string;
+
+  @Column({ type: 'varchar', length: 255 })
+  adminUserName: string;
+
+  @Column({ type: 'varchar', length: 255 })
+  adminEmail: string;
+
+  @Column('text', { array: true, default: [] })
+  activeUsers: string[];
+
+  @Column('text', { array: true, default: [] })
+  inactiveUsers: string[];
```

Comment:
PullProofAI

Consider adding validation decorators for required fields like `organizationName`, `adminUserName`, and `adminEmail`.
```diff
+  // Add validation decorators for required fields like organizationName, adminUserName, and adminEmail
+  // Example: @IsNotEmpty(), @IsString()
```


<!-- This is an auto-generated comment by PullProofAI -->